prow/plugins/trigger/ic.go

```diff
@@ -109,7 +109,7 @@ func handleIC(c client, trigger *plugins.Trigger, ic github.IssueCommentEvent) e
 		if err != nil {
 			return fmt.Errorf("error listing issue comments: %v", err)
 		}
-		trusted, err := trustedPullRequest(c.GitHubClient, trigger, commentAuthor, org, repo, comments)
+		trusted, err := trustedPullRequest(c.GitHubClient, trigger, ic.Issue.User.Login, org, repo, comments)
```

Comment:
Side note for future consideration:
 Should we be do this for _all_ kinds of testing? `/retest` is limited to tests that failed and were already requested, but with `/test foo` you could spam requesting extra testing without even being a member..? /shrug

Reply:
hypothetical problem thus far

Reply:
I mean so is malicious PR contents as far as I know. What's the use case for an untrusted commenter to trigger arbitrary tests?

Reply:
It is about simpler code, not enabling use cases

Reply:
The fact that something is technically more correct does not always make it better, if it requires higher implementation and maintenance costs.

I've seen a couple PRs that looked sketchy. And the risk/cost is a lot higher in the malicious PR scenario. Someone spamming `/test foo` doesn't really do anything aside from delay merges.